onboard/nav/searcher.hpp

```diff
@@ -42,6 +41,13 @@ class Searcher {
     NavState executeDriveToBall( Rover* phoebe );
 
 protected:  // TODO
+    /*************************************************************************/
+    /* Protected Member Functions */
+    /*************************************************************************/
+
+    //function for correcting distances between points
```

Comment:
Our function comments typically go in the .cpp files (also start function comments with verb not 'function for ...')